topical-ownership.md

```diff
@@ -0,0 +1,21 @@
+| --------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| Authentication              | Scott Seaward ([@sbwsg](https://github.com/sbwsg)), Google                       | Vincent Deemester ([@vdemeester](https://github.com/vdemeester)), Red Hat |                                                                        |
+| Bundles                     | Vincent Deemester ([@vdemeester](https://github.com/vdemeester)), Red Hat        | Pierre Tasci ([@pierretasci](https://github.com/pierretasci)), Apple      |                                                                        |
+| Conditional Execution       | Jerop Kipruto ([@jerop](https://github.com/jerop)), Google                       | Priti Desai ([@pritidesai](https://github.com/pritidesai)), IBM           |                                                                        |
+| Custom Tasks                | Jason Hall ([@ImJasonH](https://github.com/ImJasonH)), Google                    | Andrea Frittoli ([@afrittoli](https://github.com/afrittoli)), IBM         |                                                                        |
+| Events                      | Andrea Frittoli ([@afrittoli](https://github.com/afrittoli)), IBM                | Christie Wilson ([@bobcatfish](https://github.com/bobcatfish)), Google    |                                                                        |
+| Finally Tasks               | Priti Desai ([@pritidesai](https://github.com/pritidesai)), IBM                  | Christie Wilson ([@bobcatfish](https://github.com/bobcatfish)), Google    | Jerop Kipruto ([@jerop](https://github.com/jerop)), Google             |
+| PipelineResources           | Jerop Kipruto ([@jerop](https://github.com/jerop)), Google                       | Vincent Deemester ([@vdemeester](https://github.com/vdemeester)), Red Hat |                                                                        |
+| Parameters                  | Vincent Deemester ([@vdemeester](https://github.com/vdemeester)), Red Hat        | Priti Desai ([@pritidesai](https://github.com/pritidesai)), IBM           | Napoleon Santana ([@popcor255](https://github.com/popcor255)), IBM     |
+| Results                     | Priti Desai ([@pritidesai](https://github.com/pritidesai)), IBM                  | Scott Seaward ([@sbwsg](https://github.com/sbwsg)), Google                |                                                                        |
+| Variables                   | Vincent Deemester ([@vdemeester](https://github.com/vdemeester)), Red Hat        |                                                                           | Scott Seaward ([@sbwsg](https://github.com/sbwsg)), Google             |
```

Comment:
I'm happy to switch from Collaborator to Secondary here.  I initially didn't add my name there because I didn't want to hog too many spots but if noone else wants to be then I'd like to.

Reply:
sounds good, I've switched you to secondary, thank you!